data.sql

```diff
@@ -11,4 +11,45 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered,weight_kg) V
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered,weight_kg) VALUES('Angemon', '2005-06-12', 1, true, -45);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered,weight_kg) VALUES('Boarmon', '2005-06-07', 7, true, 20.4);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered,weight_kg) VALUES('Blossom', '1998-10-13', 3, true, 17);
-INSERT INTO animals (name, date_of_birth, escape_attempts, neutered,weight_kg) VALUES('Ditto', '2022-05-14', 4, true, 22);
\ No newline at end of file
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered,weight_kg) VALUES('Ditto', '2022-05-14', 4, true, 22);
+
+/* Day 3 data entry into owners table */
+
+INSERT INTO owners (full_name, age) VALUES('Sam Smith', 34);
+INSERT INTO owners (full_name, age) VALUES('Jennifier Orwell', 19);
+INSERT INTO owners (full_name, age) VALUES('Bob', 45);
+INSERT INTO owners (full_name, age) VALUES('Melody Pond', 77);
+INSERT INTO owners (full_name, age) VALUES('Dean Winchester', 14);
+INSERT INTO owners (full_name, age) VALUES('Jodie Whittaker', 38);
+
+INSERT INTO species (name) VALUES('Pokemon');
+INSERT INTO species (name) VALUES('Digimon');
+
+UPDATE animals 
+SET species_id = 1
+WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species_id = 2
+WHERE name NOT LIKE '%mon';
```

Comment:
- Kindly note that animals with names ending with `mon` are expected to belong to the `Digimon` species. According to the arrangement on lines 25 and 26, kindly alter the `species_id` on lines 29 and 33 to the right `id`. 😁